app/src/components/DisasterSearch.vue

```diff
@@ -84,7 +84,7 @@ export default {
     */
     update () {
       if (!this.query) return this.reset()
-      if (/^\d+$/.test(this.query) && this.query.length < 4) return
+      if (/^\d+$/.test(this.query) && this.query < 1000) return

```

Comment:
Two questions:

Why `this.query` on this line, but `this.query.length` on the next?

What's with `< 1000`, instead of `< 44`? These tests don't seem to match the comment at all.

Reply:
I'm changing the comment to better match the code.  We needed to test the number had 4 digits, not that it had a length of 4.  It would have returned out of the function with a query of DR, whereas it should have continued the logic, till we fixed it.

Reply:
D'oh—sorry about the `44` typo there. That's an interesting distinction—thanks for the explanation!